config/config.yml

```diff
@@ -7,7 +7,7 @@
 email:
   from: "portus@example.com"
   name: "Portus"
-  reply_to: "no-reply@example.com"
+  reply_to: "no-reply@mycompany.com"
```

Comment:
I think you should leave this empty since it's optional. Leaving a comment explaining why this is useful (or not) would be nice too.